landing_page/Gemfile.lock

```diff
@@ -78,8 +78,10 @@ GEM
       railties (>= 4.1.0)
       responders
       warden (~> 1.2.3)
+    diff-lcs (1.3)
```

Comment:
When trying to install the dependencies with `bundle install` I got this error:

```bash
Bundler could not find compatible versions for gem "method_source":
  In snapshot (Gemfile.lock):
    method_source (= 1.0.0)

  In Gemfile:
    pry-nav was resolved to 0.3.0, which depends on
      pry (>= 0.9.10, < 0.13.0) was resolved to 0.12.2, which depends on
        method_source (~> 0.9.0)

    devise was resolved to 4.7.1, which depends on
      responders was resolved to 3.0.0, which depends on
        railties (>= 5.0) was resolved to 6.0.2.2, which depends on
          method_source

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Running `bundle update` fixed it, but left me with a modified `Gemfile.lock`. It seems that the Gemfile ciuld use some tidying up. Consider this, if I have not done any code change, just pulled in the latest from the repo, there should be no changes to any file just by installing the dependencies.

Reply:
Resolved in commit d8d58c516f262cbd675e9faa70f4fbc4bfafee01